source/main.css

```diff
@@ -146,3 +146,10 @@ h6 {
     font-weight: normal;
     margin: 0;
 }
+
+@media screen and (min-width:960px){
+    body{
+        width: 960px;
+        margin: 0px auto;
```

Comment:
This is pretty good, but wouldn't it be easier to just set `max-width: 960px;` on the `body` selector at the top of this CSS? You could also put the `margin: 0 auto;` [note that you don't need the `px` unit for a value of `0`] there … wouldn't effect any screens that are smaller than `960px`.